lib/src/command/lish.dart

```diff
@@ -94,7 +94,8 @@ class LishCommand extends PubCommand {
     try {
       await log.progress('Uploading', () async {
         var newUri = host.resolve('api/packages/versions/new');
-        var response = await client.get(newUri, headers: pubApiHeaders);
+        var response = await client.get(newUri,
```

Comment:
Don't we need to add retries here and in line 124 ?

Reply:
My understanding from talking to @jonasfj was that retries might be unnecessary for the publishing process. But I do see how they'd be useful, especially as we start working towards automated publishes.